libs/wingsdk/src/core/tree.ts

```diff
@@ -134,6 +165,21 @@ function synthAttributes(
   return undefined;
 }
 
+function isIResource(construct: IConstruct): construct is IResource {
+  return construct instanceof Resource || construct instanceof Inflight;
```

Comment:
Meh... @Chriscbr what do you think about this? Not ideal... but I guess `Inflight` is going away anyway, right?

Reply:
Yeah, this looks okay to me. 🙂 BTW we could also opt for duck typing:

```ts
function isDisplayable(construct: IConstruct): boolean {
  if (construct.display) {
    return true;
  }
  return false;
}
```

Reply:
^ That would need either ts-ignore or a cast, right?

Reply:
I think we can let it as it is. What do you think?

Reply:
Sounds good